packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -715,6 +715,13 @@ describe('plugin-meetings', () => {
               height: {
                 max: 200,
                 ideal: 200
+              },
+              frameRate: {
```

Comment:
did not see this let not put a default rate here

Reply:
Keeping this as this is a test case change